backend/poetry.lock

```diff
@@ -1,10 +1,9 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
backend dockerfile should be modified to so it forces an install of poetry>=1.5.1 since for me it kept installing 1.0.11 or something outdated, so it could not build from scratch

`RUN pip install --upgrade pip && pip install poetry>=1.5.1`

Reply:
can you take over this change, make the necessary change you need